native/src/modules/categories/components/CategoryListContent.js

```diff
@@ -155,11 +134,10 @@ const CategoryListContent = ({
 
   return <SpaceBetween>
     <Container>
-      <Html html={content}
+      <Html source={{ content }}
             onLinkPress={onLinkPress}
-            contentWidth={width}
-            allowFontScaling
-            textSelectable
+            contentWidth={useWindowDimensions().width}
```

Comment:
Put the `useWindowDimensions` to the top of the component. This is better style and also the app would crash if in one render this line is called and in an other not.